src/components/share-buttons.jsx

```diff
@@ -7,56 +7,39 @@ import {
   faTwitter,
 } from '@fortawesome/free-brands-svg-icons';
 
-export default function ShearButtons() {
-  function openPopup(e, type) {
-    if (e.type === 'click' || e.keyCode === 13) {
-      e.preventDefault();
-      let url = '';
-      switch (type) {
-        case 'fb':
-          url = 'https://www.facebook.com/sharer/sharer.php?u=';
-          break;
-        case 'li':
-          url = 'https://www.linkedin.com/sharing/share-offsite/?url=';
-          break;
-        case 'twitter':
-          url = 'https://twitter.com/share?url=';
-          break;
-        default:
-          url = '';
-      }
-      /* eslint-env browser, node */
-      return window.open(url + window.location.href, '_blank');
-    }
-    return true;
-  }
+export default function ShareButtons() {
+  /* eslint-env browser, node */
```

Comment:
What is this necessary for?

Reply:
the eslint is complaining from window 
<img width="1423" alt="Screenshot 2019-10-02 at 11 19 27" src="https://user-images.githubusercontent.com/37541400/66033003-b2922f00-e506-11e9-86f1-b41ee30cc40e.png">

Reply:
We don't have `window` in our pre-rendering pipeline. When Gatsby builds our App there is browsers. You'll need to write code depending on `window`, assuming it's not always there.

In this case we can probably disable the whole component, because we cannot share without a browser.

eg.

```js
 export default function ShareButtons() { 
  if (!window) {
    return <></>
  }
  /* eslint-env browser, node */
  // …
}
```

Reply:
(deleted previous comment)

Ok but it makes no sense to add this ESLint directive to this file only. This can apply to all of our App.

You can add this to `.eslintrc.json`.

```json
{
  "env": {
    "browser": true,
    "node": true
  }
}
```

Reply:
added thanks for the help